classes/class-ledyer-templates.php

```diff
@@ -21,7 +21,12 @@ class Templates {
 
 	public function filters() {
 		// Override template if Ledyer Checkout page.
-		\add_filter( 'wc_get_template', array( $this, 'override_template' ), 999, 2 );
+
+		$settings = get_option( 'woocommerce_lco_settings' );
```

Comment:
I suggest defaulting to an empty array if the option could not be retrieved:

```
get_option( 'woocommerce_lco_settings', array() );
```

This way, when we check if an array key exists, it won't generate a warning since `get_option` default to `false`.